docs/conf.py

```diff
@@ -23,7 +23,7 @@
 author = "Huawei Noah's Ark Lab."
 
 # The full version, including alpha/beta/rc tags
-release = "0.3.6"
+release = "0.4.14"
```

Comment:
Should this be "0.4.15" now?
```suggestion
release = "0.4.15"
```